src/App.js

```diff
@@ -1,11 +1,69 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./components/pokemoncard/PokemonCard"
+import pokemonlogo from './assets/pokemonlogo.png';
+import Header from "./components/header/Header"
+import Button from "./components/button/Button"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+
+    const [ error, setError ] = useState('');
+    const [ pokemon, setPokemon ] = useState ( [])
```

Comment:
Zodra de setter aangeroepen is, dan is `pokemon` niet meer van het type array. Ik zou daarom de default waarde `undefined` maken
```suggestion
    const [pokemon, setPokemon] = useState();
```